setup.py

```diff
@@ -65,6 +65,7 @@ def find_version(*parts):
               'sphinxcontrib_github_alt',
           ],
           'test': [
+              'dask[array]',
```

Comment:
Just out of curiosity: what does that mean/do?

Reply:
Dask defines a number of 'extras', i.e. optional sets of dependencies. So just installing `dask` gets you the core library, and omits some other packages it needs for array support.

I've put it in as a test requirement, rather than a full requirement of karabo_data, because I'm wary of loading it up with too many requirements, and it seems reasonable that users who explicitly ask for a dask array will ensure dask is installed.

Reply:
Interesting, I did not know that